allennlp/common/util.py

```diff
@@ -636,3 +640,12 @@ def format_size(size: int) -> str:
     if KBs >= 1:
         return f"{round(KBs, 1):.1f}K"
     return f"{size}B"
+
+
+def hash_object(o: Any, length: int = 32) -> str:
+    """Returns a 32-character hash code of arbitrary Python objects."""
+    with io.BytesIO() as buffer:
+        pickle.dump(o, buffer)
+        hash = mmh3.hash_bytes(buffer.getvalue(), x64arch=True)
```

Comment:
Why not just use [hashlib](https://docs.python.org/3/library/hashlib.html) from the standard library?

Reply:
Fair point. That code is copy and paste from earlier projects. I changed it to use hashlib.